### PR TITLE
Polish the instructions based on what we learned

### PR DESCRIPTION
**Pull request by oliviertassinari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,31 +85,37 @@ _Developers rarely spend the time to explain the pain point they face in detail
 
 Hi, I'm facing problem, please help.
 
-#### Steps to Reproduce 🕹
+#### Steps to reproduce 🕹
 
 1. Open https://codesandbox.io/s/recursing-mclean-2dub0?file=/demo.tsx
 2. Type <kbd>1</kbd> in the textbox. Once the options are loaded, the component displays options filtered by input value. The callback `onHighlightChange` log correct value `option 1 1`.
-3. Type <kbd>2</kbd> in the textbox. The component displays options filtered by input value. The callback `onHighlightChange` log wrong value `option 1 1` instead of `option 2 12`.
+3. Then type <kbd>2</kbd> in the textbox. The textbox now contains `12`. The component displays options filtered by input value. The callback `onHighlightChange` log wrong value `option 1 1` instead of `option 2 12`.
```
Comment (m4theushw) [inline README.md:92]:
```suggestion
3. Then type <kbd>2</kbd> in the textbox. The textbox now contains `12`. The component displays options filtered by input value. The callback `onHighlightChange` logs wrong value `option 1 1` instead of `option 2 12`.
```

File: README.md [hunk 2]
```diff
@@ -81,31 +85,37 @@ _Developers rarely spend the time to explain the pain point they face in detail
 
 Hi, I'm facing problem, please help.
 
-#### Steps to Reproduce 🕹
+#### Steps to reproduce 🕹
 
 1. Open https://codesandbox.io/s/recursing-mclean-2dub0?file=/demo.tsx
 2. Type <kbd>1</kbd> in the textbox. Once the options are loaded, the component displays options filtered by input value. The callback `onHighlightChange` log correct value `option 1 1`.
```
Comment (m4theushw) [inline README.md:91]:
```suggestion
2. Type <kbd>1</kbd> in the textbox. Once the options are loaded, the component displays options filtered by input value. The callback `onHighlightChange` logs correct value `option 1 1`.
```

File: README.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ The goal of this first phase is to implement the above component (with low-level
 - [ ] make the existing test pass, add tests for edge cases
 - [ ] has no lintint errors (`yarn prettier && yarn lint && yarn typescript`)
 - [ ] has an ergonomic API
+- [ ] not using `<datalist>`
```
Comment (m4theushw) [inline README.md:40]: Do you agree? I saw `ClickAwayListener` being used. I think `makeStyles` is allowed, not sure if we should list it since it's not a component.

```suggestion
- [ ] not using `<datalist>` or any component from MUI Core
```

**Comment by DanailH** [comment]
Should we merge those? I want to send the challenge to a potential candidate but it would be best if the changes are applied before I do.

File: README.md (hunk 1 above)
Comment (oliviertassinari) [reply README.md:92]: This typo was done on purpose, to simulate how non-native English speakers developer might write their issues 😁

File: README.md (hunk 3 above)
Comment (oliviertassinari) [reply README.md:40]: Sure, I would then broaden it to:
```suggestion
- [ ] not using `<datalist>` or any external React component
```
